docs/css/extra.css

```
@@ -0,0 +1,128 @@
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'Rubik', sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Rubik" (font-family-name-quotes)

(font-family-name-quotes)